### DISCUSSION TITLE
Galaxy brain badge 2

### DISCUSSION THREAD

**Discussion by nel0x** [Schweinepriester/github-profile-achievements#58] [Q&A] [answered]
Lorem ipsum second

**Comment by melvincwng** [comment, marked as answer]
@nel0x, Hi Nicko, are you trying to get the Galaxy brain badge?

I've noticed you had a couple of answered discussions already, yet aren't getting the badge.

I might be able to provide some insights on why this is happening.

Do let me know if you need help.

**Reply by nel0x** [reply]
Hi @melvincwng! Yes, im trying - unsuccessfully - to get the Galaxy brain badge.
Also see https://github.com/Schweinepriester/github-profile-achievements/discussions/45#discussioncomment-3022202

Don't know why but - thought - I've some answered discussions (which were mostly not created by me...) in my highlights, my profile doesn't get the Brain achievement which is really strange.

**Reply by melvincwng** [reply]
@nel0x, there's a current bug whereby if you have marked your own questions as answers... You would not be able to get the badge.

My suggestion would be to look back at all your discussions, kindly unmark those questions owned by you which you have marked as answer.

Only answers marked by other users can be counted in getting the badge.

Once you cleaned up your marked answers, you would then need to get one more marked answer (by another user), then the badge should appear.

**Reply by melvincwng** [reply]
@nel0x, this is a workaround solution based on my observation.

**Reply by melvincwng** [reply]
TLDR @nel0x, basically you cannot mark any of your own questions...

If you do...the badge wouldn't appear.

**Reply by duyplus** [reply]
@melvincwng thank you <3

**Reply by melvincwng** [reply]
@duyplus, the pleasure is mine in being able to help you ^^

If you like to stay in touch, feel free to follow me on GitHub :)

Trying to expand my connections on GitHub :P

**Comment by nel0x** [comment]
Hey @melvincwng, thank you very much! You saved me lots of headaches.
For everyone else experiencing that problem: You have to go through your history and unmark all of your own answers!

**Reply by melvincwng** [reply]
@nel0x, it's my pleasure in helping fellow members of the GitHub Community.

I see in your profile page, you have unlocked the 'Galaxy Brain' badge now. Congratulations 🎉👏😊.

Moving forward, kindly do not mark your own answers to your own questions. Doing so will prevent you from unlocking higher tiers of this badge.

Take care, and have a nice day ahead 👍

**Reply by Schweinepriester** [reply]
Oh wow, funky, thank you both for discovering/debugging this!

**Reply by melvincwng** [reply]
@Schweinepriester, the pleasure is mine in helping.

Thank you too for creating this public repository that showcases all possible achievements that can be obtained in GitHub.

Hope to see future updates in this repo (assuming if more achievements will be added in the future).

**Reply by Schweinepriester** [reply]
> Hope to see future updates in this repo (assuming if more achievements will be added in the future).

For sure!
You probably know already, but in [the announcement post](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) they have confirmed as much
> We’ll only ship a few to start, but as we roll out more over time

> We’ve got more achievements up our sleeve that we’ll be releasing soon, so stay on the lookout.

For now there is still work to be done of course as per the issues open currently - the 100 % achievements as SVGs including the animation is important to me, hope to get them done soon-ish 0:-)

**Comment by NexON39** [comment]
Badgeeee ;)

**Reply by trungtaxp** [reply]
answer
